transformer_engine/common/fused_attn/fused_attn_f16_arbitrary_seqlen.cu

```diff
@@ -1185,7 +1199,7 @@ void fused_attn_arbitrary_seqlen_fwd_qkvpacked(
 
     // QKV shape is [b, s, 3, h, d]
     void *devPtrQKV = input_QKV->data.dptr;
-    const auto stride = num_head * head_dim;
+    const auto stride = 2 * num_head * head_dim;
```

Comment:
Just to clarify, we are assuming that the input data is fp16/bf16?

Reply:
Yes, these two versions of fused_attn (m512 and arbitrary) would only run with FP16/BF16 data.